test/utils/common.js

```diff
@@ -0,0 +1,89 @@
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+const template = require('lodash.template');
+const exquisite = require('exquisite-sst');
+
+let testsDir = '';
+const testFile = 'test.js';
+
+function loadFiles(directory) {
+    testsDir = directory;
+    return glob.sync(path.join(directory, '**', testFile));
```

Comment:
what does this function do?

Reply:
This funciton loads all the files for the tests recursively.